### ISSUE TITLE
Assertion failure with timeout

### ISSUE THREAD

**Issue by pjljvandelaar** [CVC4/CVC4#3329]
On windows with 
[cvc4Timer.txt](https://github.com/CVC4/CVC4/files/3671753/cvc4Timer.txt)
I get for
`cvc4 --lang=smt2.5 --incremental --strings-exp --fmf-fun-rlv --uf-ss-fair --no-strings-print-
ascii --tlimit=2000 .\cvc4Timer.txt`
the following output
```
unknown
unknown
(error "Assertion failure.
void CVC4::ResourceManager::beginCall()
/home/nightly/CVC4-master/src/util/resource_manager.cpp:216:

  d_cumulativeTimer.on()
")
```
Although the output is officially correct (an error is allowed),
I hope it is still worthwhile to report, since assertions should never fail!

**Comment by 4tXJ7f** [comment]
Yes, we don't want assertions to fail. We will take a look at the issue and let you know when we've resolved it.
